src/main/pg/motor.c

```diff
@@ -56,7 +56,16 @@ void pgResetFn_motorConfig(motorConfig_t *motorConfig)
 #else
     motorConfig->minthrottle = 1070;
     motorConfig->dev.motorPwmRate = BRUSHLESS_MOTORS_PWM_RATE;
+
+#ifdef USE_DSHOT
+#ifdef STM32F4
```

Comment:
DSHOT300 for F411 only. Also please remove this MCU specific define. This should be set in target.h for F411

`#define DEFAULT_MOTOR_DSHOT_SPEED PWM_TYPE_DSHOT300`

Then that can be used here. e.g. 
```
#ifdef USE_DSHOT
#ifdef DEFAULT_MOTOR_DSHOT_SPEED
  motorConfig->dev.motorPwmProtocol = DEFAULT_MOTOR_DSHOT_SPEED;
#else 
  motorConfig->dev.motorPwmProtocol = PWM_TYPE_DSHOT600;
#endif
#endif
```

Reply:
> Not sure why F405 should be 300... be good to have a reason.

How high can the cpu load be in the normal case (configured fc connected to the computer).
Normally, my cpu load is around 75%.

Reply:
See also: https://betaflight.com/docs/manufacturer/manufacturer-design-guidelines#41-rated-looptime-and-performance

Generally it also depends on having a BMI IMU and for F405 DS600 is only recommended for non bidirectional DS.

We also have in config.h
```
//TODO #define DSHOT_BIDIR ON
//TODO #define MOTOR_PWM_PROTOCOL DSHOT300
//TODO #define DSHOT_IDLE_VALUE 600
```
This is becoming related to trying to catch the gyro sensor type upon boot for setting the correct PID DENOM it works for BMI270 but not for ICM42688P. Because multiple gyro's are assigned to config.h a simple macro won't suffice.